tests/Kernel/ContentRenderTestBase.php

```diff
@@ -180,4 +184,59 @@ protected function createMediaImage(string $name): MediaInterface {
     return $media;
   }
 
+  /**
+   * Creates Contact entity.
+   *
+   * @param string $name
+   *   Entity name. Is used as a parameter for test data.
+   * @param string $bundle
+   *   Entity bundle.
+   * @param int $status
+   *   Entity status.
+   *
+   * @return \Drupal\oe_content_entity_contact\Entity\ContactInterface
+   *   Contact entity.
+   */
+  protected function createContactEntity(string $name, string $bundle, int $status = CorporateEntityInterface::PUBLISHED): ContactInterface {
```

Comment:
Rather than duplicating this code should we not add this in a trait?

Reply:
createMediaImage() and creation of another entities can be also placed to traits. We have started to refactor tests but then it was stopped.

Please discuss with @ademarco.